src/content/docs/pt-br/basics/astro-pages.mdx

```diff
@@ -97,3 +100,75 @@ Para uma página customizada de erro 404, você pode criar um arquivo `404.astro
 
 Isso irá construir uma página `404.html`. A maioria dos [serviços de deploy](/pt-br/guides/deploy/) irão a encontrar e utilizar.
 
+## Parciais de páginas
+
+<p><Since v="3.4.0" /></p>
+
+:::caution
+É esperado que parciais de páginas sejam utilizadas em conjunto com uma biblioteca front-end, como [htmx](https://htmx.org/) ou [Unpoly](https://unpoly.com/). Você também pode utiliza-las caso se sinta confortável escrevendo JavaScript front-end low-level. Por conta disso elas são consideradas um recurso avançado.
+
+Adicionalmente, parciais não devem ser usadas se o componente possuir estilos com escopo ou scripts, pois esses elementos serão removidos do HTML gerado. Se você precisa de estilos com escopo, melhor utilizar páginas padrões sem parciais de páginas em conjunto com uma biblioteca frontend que sabe mesclar conteúdo no head.
+:::
+
+Parciais são componentes de página localizados em `src/pages/` que não tem como objetivo serem renderizados em páginas completas.
+
+Assim como componentes localizados fora dessa pasta, esses arquivos não incluem a declaração `<!DOCTYPE html>` automaticamente, nem qualquer outro conteúdo do `<head>`, como estilos com escopo ou scripts.
+
+Porém, por estarem localizados no diretório especial `src/pages/`, o código HTML gerado fica disponível em uma URL que corresponde ao caminho do arquivo. Isso permite que uma biblioteca de renderização(ex: htmx, Stimulus, jQuery) possa acessa-lo no cliente e carregar sessões do HTML dinamicamente em uma página, sem a necessecidade de um recarregamento ou navegação entre páginas.
+
+Parciais, quando combinados com uma biblioteca de renderização, proporcionam uma alternativa às [Ilhas Astro](/pt-br/concepts/islands/) e [tags `<script>`](/pt-br/guides/client-side-scripts/) na construção de conteúdo dinâmico em Astro.
+
+Arquivos de página que podem exportar um valor(ex: `.astro`, `.mdx`) podem ser marcadas como parciais.
+
+Configure um arquivo no diretório `src/pages/` para ser uma parcial adicionando a seguinte exportação:
+
+```astro title="src/pages/parcial.astro" ins={2}
+---
+export const partial = true;
+---
+
+<li>Eu sou uma parcial!</li>
+```
+
+O `export const partial` deve ser identificável estaticamente. Ele pode conter os seguintes valores.
+
+- O booleano __`true`__.
+- Uma variável de ambiente usando import.meta.env como `import.meta.env.USE_PARTIALS`.
+
+### Usando com uma biblioteca
+
+Parciais são usadas para atualizar, dinamicamente, sessões de uma página, usando bibliotecas como [htmx](https://htmx.org/).
```

Comment:
I don't think the commas are correct here.

```suggestion
Parciais são usadas para atualizar dinamicamente sessões de uma página usando bibliotecas como [htmx](https://htmx.org/).
```

The first two are delimiting a qualifier, this is used when the qualifier is not making the previous term more specific.
- "Humanos, que são animais, ..." All humans are animals, so that can be between commas.
- "Atualizações dinâmicas ... Essas formas de atualizar, dinâmicamente, ..." The second use is already referring to the first, the methods in questions are about dynamic updates, so that can also be between commas.

That is not the case here since it is the first sentence of the section and not all updates are dynamic.

I can't think of a scenario that would make the third comma correct with the same words.